packages/fx-core/src/component/driver/arm/util/util.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import path from "path";
+import { TemplateType } from "../constant";
+import { deploymentOutput, templateArgs } from "../interface";
+import { DriverContext } from "../../../interface/commonArgs";
+
+export function hasBicepTemplate(args: templateArgs[]): boolean {
+  for (const arg of args) {
+    const templateType = path.extname(arg.path).toLowerCase();
+    if (templateType === TemplateType.Bicep) {
+      return true;
+    }
+  }
+  return false;
+}
+
+// TODO: should update when context get path property
+export function getPath(path: string, context: DriverContext): string {
+  return path;
+}
+
+export function convertOutputs(outputs: deploymentOutput[]): Map<string, string> {
+  const res = new Map<string, string>();
+  for (const output of outputs) {
+    convertOutput(output, res);
+  }
+  return res;
+}
+
+function convertOutput(output: deploymentOutput, map: Map<string, string>, prefix?: string) {
```

Comment:
minor: may add a comment to describe the naming conversion rule for the output keys